src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorLivePreviewWidget.ts

```diff
@@ -74,6 +74,7 @@ export class InteractiveEditorLivePreviewWidget extends ZoneWidget {
 			diffCodeLens: false,
 			stickyScroll: { enabled: false },
 			minimap: { enabled: false },
+			isInEmbeddedEditor: true
```

Comment:
I believe we need a similar change in [InteractiveEditorFileCreatePreviewWidget](https://github.com/microsoft/vscode/blob/dc897c6c4fa6e9eecc98c70e4931dbdc16a4027c/src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorLivePreviewWidget.ts#L331) which is the embedded preview editor for file creation. It's a normal, embedded editor tho

Reply:
Tested the preview editor and arrow keys work fine there, one of the context keys might already take care of that.